### PR TITLE
Translation of the lifting-state-up in Turkish

### PR DESCRIPTION
**Pull request by Poseid10ur**

### CODE REVIEW

File: content/docs/lifting-state-up.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: lifting-state-up
-title: Lifting State Up
+title: State'i Ebeveyne Taşıma
```
Comment (alioguzhan) [inline content/docs/lifting-state-up.md]: What about `State i Yukari Tasima` ? How is this ?

**Comment by alioguzhan** [comment]
`Parent` and `Child` words need to be translated as `üst` and `alt`. see #21 

And (just a recommendation), possible translation for this section's title would be `State'i Yukarı Taşımak` or `State'i Üst Elemana Taşımak`

We can discuss it, though.

**Comment by Poseid10ur** [comment]
@alioguzhan I believe 'ortak üst eleman' will be more clear in 'lifting-state-up' context. Because 'üst' might create ambiguity in this context.

**Comment by Poseid10ur** [comment]
@alioguzhan , I changed the title and some of the context phrases which means parent. You can have a look if you want.

**Comment by alioguzhan** [comment]
Hmm. I think we need to behave differently to title and content. The updated version of the title looks better to me now. (`State'i Yukari Tasima`)

But in the content section, `en yakin ortak ust eleman` for `closest common ancestor` looks way better. What do you think?

**Comment by Poseid10ur** [comment]
@alioguzhan yes, you're right. `en yakin ortak ust eleman` will be better choice for the `closest common ancestor`. Applied the required changes. Thanks!

**Pull request merged**